textdb/textdb-common/src/main/java/edu/uci/ics/textdb/common/constants/SchemaConstants.java

```diff
@@ -15,9 +15,11 @@
     public static final String SPAN_END = "spanEnd";

```

Comment:
I saw you already have many places  using the original order.  I hope it's not hard to make those changes.